code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -229,22 +231,30 @@
             [:<>
              (when (is-old-version? nb-version)
                [ui/Message
-                {:warning true
+                {:error   true
                  :icon    {:name "warning sign", :size "large"}
                  :header  (@tr [:nuvlabox-update-warning])
                  :content (r/as-element
-                            [:span (str (@tr [:nuvlabox-update-warning-content])) " "
+                            [:span (str (@tr [:nuvlabox-update-error-content])) " "
                              [:a {:href   (str "https://docs.nuvla.io/nuvlabox/"
                                                "nuvlabox-engine/quickstart.html#from-nuvla")
                                   :target "_blank"}
                               (str/capitalize (@tr [:see-more]))]])}])
+             (when (and (not (nil? target-version)) (is-old-version? target-version))
```

Comment:
there is a function named "some?" 
(defn some? [x] (not (nil? x)))
https://github.com/clojure/clojure/blob/clojure-1.10.1/src/clj/clojure/core.clj#L536